lib/src/api.mli

```diff
@@ -127,7 +125,9 @@ module type BASIC_STRING = sig
   (** Test whether a string is empty. *)
 
   val make: int -> character -> t
-  (** Build a new string like [String.make]. *)
+  (** Build a new string like [String.make].
+
+   @raise Invalid_argument if size is [< 0] or TODO *)
```

Comment:
Because the underlying one also raises an exception if `size` is greater than `Sys.max_string_length`, which we need to mirror :smile: